xlsxconverter/XLSXConverter2.js

```diff
@@ -426,11 +426,16 @@ var XLSXConverter = {};
 			// odk time stamp -- UTC timestamp string with 00:00:00.000 time.
             "type": "string",
 			"elementType": "date"
+        },
+        "dateshort": {
+            // odk date with no time or timezone information -- stored as YYYY / MM / DD in the Gregorian calendar
+            "type": "string",
+            "elementType": "dateShort"
         },
 		"birthdate": {
 			// odk time stamp -- UTC timestamp string with 00:00:00.000 time.
             "type": "string",
-			"elementType": "date"
+			"elementType": "dateShort"
```

Comment:
Should we add a new `birthdateshort` type instead? So we don't break existing users of the `birthdate` type.

Reply:
that is a good idea @linl33. Since it's not been widely released yet we can probably safely make it "birth_date" and put that in the documentation and slowly deprecate 'birthdate'

Reply:
I'm open to debate here, but my understanding after discussing with @wbrunette is that we consider the existing birthdate functionality to be buggy, and this will fix it. Your suggestion supports two parallel, presumably valid formats. IMO I'd leave this as is to fix the bug but open to disagreement. 

It is true that existing birthdates will be broken (I ran into this myself in testing). They will be decoded from the DB incorrectly. IMO if we want to add code to account for that, I would try to write a one-off special case that we can later remove that checks for the case where we are both "birthdate" and elementType="date". This is now an invalid case, but we can check for it and store the date in UTC in that case. 

cc @wbrunette what's your opinion, and do you think the special case for backwards compatibility is worth it? I don't know how long the birthdate feature has been released and whether we think anyone is using it.

Reply:
It is my understanding as well that the current `birthdate` behavior is buggy and the new type should replace it. I'm worried that naming the new type the same as the old type is going to be an obstacle for those are using the `birthdate` type right now.
Maybe we should remove the existing `birthdate` type from XLSXConverter but name the new type something different, `birth_date` like Waylon suggest or something else. This way the existing tables will continue to function while new forms can use the better implementation.